docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -140,27 +140,30 @@ Below topologies from ONNX open model zoo are fully supported on OpenVINO Execut
 
 ## CSharp API
 
-*To use csharp api for openvino execution provider create a custom nuget package.
+To use csharp api for openvino execution provider create a custom nuget package.
 
-#Windows
+1.Windows
 
-Build windows build with --build_csharp option. 
+Build a custom nuget package for windows.
+```
 .\build.bat --config Debug --build --use_openvino $Device --build_csharp
-
-Build Custom Nuget Package
 msbuild csharp\OnnxRuntime.CSharp.proj /p:Configuration=Debug /t:CreatePackage
 
-#Linux
-We do not have a current build process to build directly in Linux. But we can
+```
+
+2.Linux
+
+We currently do not have a process to build directly in Linux. But we can
 copy shared library libonnxruntime.so to onnxruntime repository in windows
```

Comment:
Can you also show the command line to generate the libonnxruntime.so in linux? Under linux, after the general description, you can show the two commands as 'on Linux machine' and 'on Windows machine' headers.

Reply:
done